observability/prometheusrules.jsonnet

```
@@ -503,6 +503,20 @@ local renderAlerts(name, environment, mixin) = {
         {
           name: 'observatorium-metrics',
           rules: [
+            {
+              alert: 'ObservatoriumNoStoreBlocksLoaded',
+              annotations: {
+                description: 'Observatorium Thanos Store {{$labels.namespace}}/{{$labels.job}} has not loaded even a single block.',
+                summary: 'Observatorium Thanos Ruler has not loaded even a sigle block. This should not have happened, indicates data loss. Check out the configuration.',
```

Maybe we could add information on how long
```suggestion
                summary: 'Observatorium Thanos Ruler has not loaded a single block in the last 6 hours.',
```